### PR TITLE
fix: Account automation report show error

### PR DESCRIPTION
**Pull request by fit2bot**

### CODE REVIEW

File: src/views/reports/accounts/AccountAutomation.vue [hunk 1]
```diff
@@ -211,7 +211,6 @@ export default {
   methods: {
     onChange(val) {
       this.days = val
-      localStorage.setItem('reportDays', val)
     },
     async getData() {
       const data = await this.$axios.get(`/api/v1/reports/reports/account-automation/?days=${this.days}`)
```
Comment (baijiangjie) [inline src/views/reports/accounts/AccountAutomation.vue:216]: I haven't been able to directly run the provided code snippet because I cannot see it. However, you can provide me with an updated version of the proposed code or a copy-paste from another file if needed. If you're asking for any general feedback regarding differences between two versions (for example before and after making modifications), please give more specifics about what you observed that might have changed, what's not working like expected, or other relevant information.

If you want specific advice based on existing functionality such as improvements in efficiency, optimizations, better UX, etc., without knowing the real context, here is some general guidance:

1. **Consistent Coding Standards**: Ensure consistent practices when formatting code. This could include proper indentation, spacing within lines using spaces instead of tabs, use of comments wisely, and naming conventions.
   
2. **Modular Structures and Libraries**: Use reusable components and libraries where appropriate. This makes the overall project simpler and faster due to less dependency management.

3. **Efficiency Measures:** Look into improving performance through techniques like memoization and caching functions for repeated computations, avoiding unnecessary database hits where possible, and optimizing data retrieval operations.

4. **Security Practices**: Always be wary of security vulnerabilities like SQL injection, XSS attacks, and insecure storage methods. Regularly update dependencies which may introduce new vulnerabilities.

5. **Maintainability**: Make sure all changes are reversible – ideally each change should result only in minor regression testing failure. Documenting these changes helps maintain them properly and also aids debugging.

6. **Testing Strategy**: Consider implementing unit tests for individual component functionalities, integration/functional tests for complex setups including edge cases, and continuous integration and delivery (CI/CD). 

Remember to review every feature being added and make sure these do not conflict with others (i.e., don’t create circular references).

If there were intended changes and the code did break or behave differently than anticipated after those changes were made, then further examination would likely reveal how they affected parts of the system.

**Review by w940853815 — APPROVED** [review]
